src/parsers/ecs-event.js

```diff
@@ -0,0 +1,75 @@
+//
+// AWS ECS Event
+//
+exports.matches = event =>
+	event.getSource() === "ecs";
+
+exports.parse = event => {
+	const time = new Date(event.get("time"));
+	const detailType = event.get("detail-type");
+	const status = event.get("detail.lastStatus");
+	const desiredStatus = event.get("detail.desiredStatus");
+	const service = event.get("detail.group");
+	const clusterArn = event.parseArn(event.get("detail.clusterArn"));
+	const clusterUrl = `https://console.aws.amazon.com/ecs/home?region=${clusterArn.region}#/${clusterArn.suffix}/services`;
```

Comment:
For me, this URL is generated: https://console.aws.amazon.com/ecs/home?region=ap-southeast-2#/undefined/services

I think this is how it should look, at least it works for me: https://console.aws.amazon.com/ecs/home?region=ap-southeast-2#/clusters/ABC/services

clusterArn: arn:aws:ecs:ap-southeast-2:999999999999:cluster/ABC